data/endpoint_data.json

```diff
@@ -0,0 +1,158 @@
+{
+    "comparison": "USA",
+    "metrics": [
+        {
+            "indicator": "GINI INDEX",
+            "ranking": 72,
+            "sortedCountries": ["Armenia", "Azerbaijan", "USA", "Bulgaria", "Georgia", "Greece", "Iran", "Iraq", "Syria"],
+            "lastPosition": {
+				"country": "Slovenia",
```

Comment:
As well as all the other references by full name.
I know this is mock data but we will need to address this throughout the logic.

Reply:
done